<ui version="4.0" >
 <class>qdFileOperation</class>
 <widget class="QDialog" name="qdFileOperation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="qlCount" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>245</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
   <property name="buddy" >
    <cstring>qcbDestination</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>64</y>
     <width>245</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Filter:</string>
   </property>
   <property name="buddy" >
    <cstring>qcbFilter</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="qcbFilter" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>84</y>
     <width>245</width>
     <height>18</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="qcbDestination" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>35</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbBrowse" >
   <property name="geometry" >
    <rect>
     <x>178</x>
     <y>32</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>4</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>&amp;Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbOK" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>109</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbEnqueue" >
   <property name="geometry" >
    <rect>
     <x>95</x>
     <y>109</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Enqueue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbCancel" >
   <property name="geometry" >
    <rect>
     <x>178</x>
     <y>109</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qpbOK</tabstop>
  <tabstop>qpbEnqueue</tabstop>
  <tabstop>qpbCancel</tabstop>
  <tabstop>qpbBrowse</tabstop>
  <tabstop>qcbDestination</tabstop>
  <tabstop>qcbFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
